client/src/templates/Challenges/utils/index.js

```diff
@@ -1,7 +1,9 @@
-const guideBase = 'https://www.freecodecamp.org/forum/search?q=';
+import { forumLocation } from '../../../../../config/env.json';
 
-export function createGuideUrl(title = '') {
-  return guideBase + title + '%20in%3Atitle%20order%3Aviews';
+export function getGuideUrl({ forumTopicId, title = '' }) {
+  return forumTopicId
```

Comment:
```suggestion
  title = encodeURIComponent(title);
  return forumTopicId
```
In case the title includes &, ? etc.